policy-library/Kptfile

```diff
@@ -0,0 +1,11 @@
+apiVersion: kpt.dev/v1alpha1
```

Comment:
Is this required? would we expect users to use kpt themselves later to refresh the policies?

Reply:
it is not not really necessary. removed